pre_test.sh

```diff
@@ -53,6 +53,10 @@ $IPDK_RECIPE/install/sbin/ovs-vswitchd --detach --no-chdir unix:$RUN_OVS/var/run
 echo "set hugepages"
 $IPDK_RECIPE/scripts/dpdk/set_hugepages.sh
 
+rm -rf /usr/share/stratum/dpdk/dpdk_skip_p4.conf
+cp common/p4c_artifacts/pna_tcp_connection_tracking/dpdk_skip_p4.conf /usr/share/stratum/dpdk/ 
```

Comment:
/usr/share/stratum/dpdk - will this path always exist? or do you want to check if it exists, if not then create it?